application/src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentController.java

```diff
@@ -99,30 +105,47 @@ void init() throws NoSuchMethodException {
                 .body(new StoredDocumentHalResource(storedDocument));
     }
 
-
-
     @GetMapping(value = "{documentId}/binary")
     @ApiOperation("Streams contents of the most recent Document Content Version associated with the Stored Document.")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Returns contents of a file")
     })
-    public ResponseEntity<Object> getBinary(@PathVariable UUID documentId) {
+    public ResponseEntity<Object> getBinary(@PathVariable UUID documentId, HttpServletResponse response) {
 
         DocumentContentVersion documentContentVersion =
-                documentContentVersionService.findMostRecentDocumentContentVersionByStoredDocumentId(documentId);
+            documentContentVersionService.findMostRecentDocumentContentVersionByStoredDocumentId(documentId);
 
         if (documentContentVersion == null || documentContentVersion.getStoredDocument().isDeleted()) {
             return ResponseEntity.notFound().build();
```

Comment:
Would you have a chance in this story to find out how documentContentVersion.getStoredDocument().isDeleted() is used and check whether we can reduce complexity to one single place

Reply:
As per chat we have had we will leave it as it is for now.